docs/UserGuide.md

```diff
@@ -427,19 +429,19 @@ Examples:
 
 Changes the displayed Staff list to contain only Staff members with names that match any of the keywords.
 
-Format: `findstaff KEYWORDS`
+Format: `findstaff KEYWORD [MORE_KEYWORDS]`
 
 ```yaml
 Note:
-  * `KEYWORDS` must not be an empty string.
-  * For `findstaff` to work as expected, the current active Staff list must be non-empty.
-  * The findstaff command is case-insensitive such that 'hans' will match 'HANS'.
-  * The findstaff command will match Staff name if there is a partial match, 'ha' or
+  - KEYWORD must not be an empty string.
+  - For `findstaff` to work as expected, the current active Staff list must be non-empty.
+  - The findstaff command is case-insensitive such that 'hans' will match 'HANS'.
+  - The findstaff command will match Staff name if there is a partial match, 'ha' or
     'hA' will both match 'HANS'.
 ```
 
 Examples:
-* `findstaff Jon` returns a filtered view of the Staff list such that all Staff with name `Jon` are listed.
+* `findstaff Jon` returns a filtered view of the Staff list such that all Staff with name `Jon` (e.g. `Jonson`, `Hajon`) are listed.
```

Comment:
maybe change to 
`findstaff Jon` returns a filtered view of the Staff list such that all Staff whose name contain `Jon` (e.g. `Jonson`, `Hajon`) are listed.